### PR TITLE
noop lint cleanup of common-hid-packet-parser

### PR DESCRIPTION
**Pull request by ywwg**
While we don't plan to make lots of changes to the common-hid script, it should at least pass the linter

### CODE REVIEW

**Review by Holzhaus — APPROVED** [review]
Thank your for this tedious cleanup.

**Comment by Holzhaus** [comment]
Can someone with a HID controller verify that this doesn't break anything?

**Comment by ywwg** [comment]
for what it's worth, it doesn't break my S3 in-progress mapping

**Comment by uklotzde** [comment]
Diff looks clean and sane. LGTM

**Comment by Be-ing** [comment]
This broke the Kontrol S2 Mk2 mapping: https://mixxx.discourse.group/t/ni-traktor-s2-mk2-hid-mapping/16856/34

Mass reformatting is dangerous.

File: res/controllers/common-hid-packet-parser.js [hunk 1]
```diff
@@ -338,101 +340,101 @@ HIDPacket.prototype.removeControl = function(group, name) {
...
  */
 HIDPacket.prototype.addControl = function(group, name, offset, pack, bitmask, isEncoder, callback) {
-    var control_group = this.getGroup(group, true)
-    var bitvector = undefined
-    if (control_group == undefined) {
-        HIDDebug("ERROR creating HID packet group " + group)
-        return
+    var control_group = this.getGroup(group, true);
+    var bitvector = undefined;
+    if (control_group === undefined) {
```
Comment (Be-ing) [inline res/controllers/common-hid-packet-parser.js:345]: Changing `==` to `===` is not a no-op!!

**Pull request merged**